### DISCUSSION TITLE
Same app for Google Play, F-droid and IzzyOnDroid lead to some confusion.

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [FossifyOrg/General-Discussion#149] [General]
Having all or atleast 2 of these repos makes updating a bit confusing. Installing an app through f-droid (f-droid repo), but then it updates the app through Google Play och IzzyOnDroid. It confuses me and i guess most normal users. I suppose this is have to do something like signing keys or like that- Whatever reason it not a optimal experiance for the end user. I suppose this is a problem best solved upstream because not just these apps are affected by this. 

However i don't think this problem is solved upsteam in the nearterm future. Anyone have any insight in this problem, @IzzySoft ? If it could be easly solved it would be great.

**Comment by naveensingh** [comment]
I sign all Fossify releases using the same key to allow seamless migration between Google Play and F-Droid versions. 

If you don't want Google Play to replace F-Droid versions, you can disable auto updates in Google Play:

<img src="https://github.com/FossifyOrg/General-Discussion/assets/36371707/8129bbdd-61c6-4a0f-83cd-79393777f84b" width=264 />

**Reply by IzzySoft** [reply]
And as for F-Droid vs. IzzyOnDroid: It's the very same APK. You'll usually get it a few days earlier via my repo (as F-Droid's build cycle takes at least 2..3 days while I pick the APK directly from Fossify), but in the end F-Droid will ship the very same APK (or nothing in case "reproducible build" failed).

A "later" version of the F-Droid client (still in development) will allow you to set the "preferred repo", so you e.g. get updates offered only from where you originally installed the app from unless you explicitly change that – which then will fully solve your issue.
